core/common/src/main/java/alluxio/client/file/CacheContext.java

```diff
@@ -43,7 +43,7 @@ public static CacheContext defaults() {
     return new CacheContext();
   }
 
-  private CacheContext() {} // prevent instantiation
+  protected CacheContext() {} // prevent instantiation
```

Comment:
can you add javadoc here saying this class is expected to be inherited?

Reply:
Javadoc added